conformance/ion_encoding/module/macro/trivial/invoke_ee.ion

```diff
@@ -0,0 +1,54 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Very simple macro definition and E-expression invocation
+
+
+// Ensure that malformed templates are signaled when they are encountered,
+// not delayed until the module is installed or macro is invoked.
+(ion_1_1
+  (encoding (module M (macro_table (macro m () ()))))
```

Comment:
The malformed-ness of this macro is because the body has a s-expression with no macro ref, right?

Reply:
Exactly!